### ISSUE TITLE
Abstract method in addressbook.php

### ISSUE THREAD

**Issue by hoodie** [owncloud/contacts#804]
Hi there,
I just updated to OC8, but unfortunately I get this
```shell
Fatal error: Class OCA\Contacts\Share\Addressbook contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (OCP\Share_Backend::isShareTypeAllowed) in /xxxpathxxx/apps/contacts/lib/share/addressbook.php on line 130 
```
php is php5.4.36-0

any ideas?

**Comment by abstractmodel** [comment]
same here. had to disable the contact and calendar apps after upgrade to OC 8.0...

**Comment by RealRancor** [comment]
Hi,

have you both updated the calendar and contacts app from the appstore as the one of OC7 is not compatible with OC8.

**Comment by hoodie** [comment]
You are right, I would have expected the newer versions of these apps to be included in owncloud8 as they were in previous versions.

**Comment by RealRancor** [comment]
Hi,

unfortunately that wasn't really communicated. There was a mailinglist post:

http://owncloud.10557.n7.nabble.com/ownCloud-8-alpha-1-td13757.html

but nothing else. But it shouldn't matter if the app is shipped or not if the following upgrade guide is followed:

http://doc.owncloud.org/server/8.0/admin_manual/maintenance/upgrade.html
